src/types/transaction.ts

```diff
@@ -46,6 +46,12 @@ export interface SendTransactionResult {
   confirmation?: modelsv2.PendingTransactionResponse
 }
 
+/** The result of sending a fund transaction from AlgoKit Dispenser API */
+export interface SendDispenserTransactionResult {
+  /** The transaction ID */
+  transaction: string
```

Comment:
This should probably be `transactionId` to be clearer

Reply:
closing, this is completely refactored now